scripts/run_e2e_tests.py

```diff
@@ -505,6 +505,24 @@ def run_e2e_tests(
                         ],
                         show_output=verbose,
                     )
+            elif basename(spec_path) == "hostframe.spec.js":
+                test_name, _ = splitext(basename(spec_path))
+                test_name, _ = splitext(test_name)
+                test_path = join(
+                    ctx.tests_dir, "scripts", "hostframe", "hostframe_app.py"
+                )
+                if os.path.exists(test_path):
+                    run_test(
+                        ctx,
+                        str(spec_path),
+                        [
+                            "streamlit",
+                            "run",
+                            "--ui.hideSidebarNav=false",
```

Comment:
I think(?) this flag (along with the one for multipage apps tests) can be removed. My guess is that it was added to the MPA tests after we had merged the changes but before they were enabled by default, and I must have forgotten to remove it after the MPA release.